src/lib/components/lines-and-dots/svg/timeline-graph-row.svelte

```diff
@@ -121,7 +121,7 @@
             config={TimelineConfig}
             icon="retry"
           >
-            {group?.displayName} • {translate('workflows.retry')}
+            {group?.displayName} • {translate('workflows.attempt')}
```

Comment:
Do we want to update `Retry Expiration` or `Next Retry` as well?

Reply:
I think those still make sense. Like, you are on Attempt 4, but your next retry is in X or will expire in X. Retry is like future attempt.